### ISSUE TITLE
PerfTests : Transports fixture history does not show lines for all permutations

### ISSUE THREAD

**Issue by ramonsmits** [Particular/EndToEnd#151]
Transports fixture history does not show lines for all permutations

The bar chart below shows ReceiveRunner SQLServer_Azure which is not visible in the line chart history

![image](https://cloud.githubusercontent.com/assets/152998/16415104/d00ba046-3d3b-11e6-9b60-3891bd26fc27.png)

**Comment by dvdstelt** [comment]
@ramonsmits Do you know how to fix this? Should we Google this? I have no idea how hard/easy this is to solve? Should we get help from someone else?

**Comment by ramonsmits** [comment]
@dvdstelt No, not at all. I do think that @WilliamBZA mentioned that its possible to show all lines. I almost never compare different transports. I usually want to compare V5 vs V6 for a specific transport and it shows all lines in that scenario.

**Comment by WilliamBZA** [comment]
Definitely possible to remove the "OTHER" line. If I recall correctly it's just a change in the query. Something like `LIMIT 40` or something.

The question is is there value in showing that many lines at once? Perhaps the query would be better to show a subset of the results.
